examples/js/effects/VREffect.js

```diff
@@ -102,16 +102,26 @@ THREE.VREffect = function ( renderer, onError ) {
 
 	document.addEventListener( fullscreenchange, function () {
 
-		isPresenting = isDeprecatedAPI && vrHMD && ( document.mozFullScreenElement || document.webkitFullscreenElement ) !== undefined;
+		isPresenting = vrHMD && ( vrHMD.isPresenting || isDeprecatedAPI && ( document.mozFullScreenElement || document.webkitFullscreenElement ) !== undefined );
```

Comment:
I agree that it's difficult to read. Maybe an `if` statement or two would be clearer. Would you prefer that?